language/sentiment/README.md

```diff
@@ -0,0 +1,48 @@
+# Introduction
+
+This sample contains the code referenced in the 
+[Sentiment Analysis Tutorial](http://cloud.google.com/natural-language/docs/sentiment-tutorial)
+within the Google Cloud Natural Language API Documentation. A full walkthrough of this sample
+is located within the documentation.
+
+This sample is a simple illustration of how to construct a sentiment analysis
+request and process a response using the API.
+
+## Prerequisites
+
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/), including the [gcloud tool](https://cloud.google.com/sdk/gcloud/), and [gcloud app component](https://cloud.google.com/sdk/gcloud-app).
+
+2. Setup the gcloud tool. This provides authentication to Google Cloud APIs and services.
+
+```
+$ gcloud init
+```
```

Comment:
Actually, I think this won't work. Does `gcloud init` set up a service account for you? IIUC, without setting up a service account, this will attempt to use the gcloud auth, which doesn't have the api enabled.